storage/reads/datatypes/storage_common.proto

```diff
@@ -24,6 +24,12 @@ service Storage {
 
   rpc Hints (google.protobuf.Empty) returns (HintsResponse);
 
+  // TagKeys performs a read operation for tag keys
+  rpc TagKeys (TagsRequest) returns (stream TagsResponse);
```

Comment:
need distinct `TagKeysRequest` and `TagValuesRequest` arguments to these APIs, as they have different parameters